pypeit/spectrographs/p200_dbsp.py

```diff
@@ -0,0 +1,422 @@
+            meta_key: str
+
+        Returns:
+            value:
+
+        """
+        retval = super(P200DBSPBlueSpectrograph, self).compound_meta(headarr, meta_key)
+        if retval is not None:
+            return retval
+        if meta_key == 'binning':
+            binspatial, binspec = headarr[0]['CCDSUM'].split(' ')
+            return parse.binning2string(binspec, binspatial)
+        else:
+            msgs.error("Not ready for this compound meta")
+
+    def get_detector_par(self, hdu: fits.HDUList, det: int):
+        """
+        Return a DectectorContainer for the current image
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`):
+                HDUList of the image of interest.
+                Ought to be the raw file, or else..
+            det (int):
+
+        Returns:
+            :class:`pypeit.images.detector_container.DetectorContainer`:
+
+        """
+        # Binning
+        binning = self.get_meta_value(self.get_headarr(hdu), 'binning')  # Could this be detector dependent??
+
+        # Detector 1
+        detector_dict = dict(
+            binning         = binning,
+            det             = 1,
+            dataext         = 0,
+            specaxis        = 0,
+            specflip        = True,
+            spatflip        = False, # check
+            platescale      = 0.389,
+            darkcurr        = 0.0,
+            saturation      = 65000.,
+            nonlinear       = 62./65.,
+            mincounts       = -1e10, # cross-check
+            numamplifiers   = 1,
+            gain            = np.atleast_1d(0.72),
+            ronoise         = np.atleast_1d(2.5)
+            )
+        
+        header = hdu[0].header
+        datasec = header['DSEC1']
+        oscansec = header['BSEC1']
+
+        detector_dict['datasec'] = np.atleast_1d(flip_fits_slice(datasec))
+        detector_dict['oscansec'] = np.atleast_1d(flip_fits_slice(oscansec))
+
+        return detector_container.DetectorContainer(**detector_dict)
+
+
+    def default_pypeit_par(self):
+        """
+        Set default parameters for P200 DBSPb reductions.
+        """
+        par = pypeitpar.PypeItPar()
+        par['rdx']['spectrograph'] = 'p200_dbsp_blue'
+
+        # Ignore PCA
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+
+
+        par['scienceframe']['process']['use_overscan'] = True
+        # Make a bad pixel mask
+        par['calibrations']['bpm_usebias'] = True
+        # Set pixel flat combination method
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
+        par['calibrations']['pixelflatframe']['process']['sig_lohi'] = [10.,10.]
+        # Change the wavelength calibration method
+        par['calibrations']['wavelengths']['method'] = 'full_template'
+        par['calibrations']['wavelengths']['reid_arxiv'] = 'p200_dbsp_blue.fits'
+        par['calibrations']['wavelengths']['lamps'] = ['FeI', 'FeII', 'ArI', 'ArII']
```

Comment:
I remember constructing the Fe I and Fe II lists somewhat haphazardly!! I just thought I'd add this as a warning if you're finding some funky wavelength calibration results!